src/Services/Generators/Gravatar.php

```diff
@@ -2,22 +2,24 @@
 
 namespace LaravelEnso\Avatars\Services\Generators;
 
-use Illuminate\Http\File;
 use Illuminate\Support\Facades\Http;
-use Illuminate\Support\Facades\Storage;
+use LaravelEnso\Avatars\Enums\Types;
 use LaravelEnso\Avatars\Models\Avatar;
 
 class Gravatar extends Generator
```

Comment:
let's drop the generator class and use constructors in both gravatar and laravolt